docs_user/modules/proc_adopting-key-manager-service.adoc

```diff
@@ -32,7 +32,6 @@ spec:
     template:
       databaseInstance: openstack
       databaseAccount: barbican
-      databaseUser: barbican
```

Comment:
@xek I removed the "databaseUser" field all together because that field is supposed to be changed to "databaseAccount". But there is already a "databaseAccount" field above it.

Reply:
The "databaseUser: barbican" also appears in the tests
tests/roles/barbican_adoption/tasks/main.yaml:20
Please also remove this line from there, this way if the tests pass, we will be sure this is correct.

Reply:
I posted a separate PR for tests at https://github.com/openstack-k8s-operators/data-plane-adoption/pull/535